apis/r/tools/install-pkgdown-dependencies.sh

```diff
@@ -5,5 +5,7 @@ export R_REMOTES_NO_ERRORS_FROM_WARNINGS='true'
 Rscript -e 'install.packages(c("remotes", "pkgdown"))'
 cd apis/r
 Rscript -e 'remotes::install_deps(dependencies = TRUE, upgrade=FALSE)'
-Rscript -e 'remotes::install_local()'
+# Need dependencies = FALSE to avoid getting latest/sometimes-too-new r-tiledb from CRAN when we
+# really want it from our r-universe where we control the version we want:
+Rscript -e 'remotes::install_local(dependencies = FALSE)'
```

Comment:
Just for completeness:  This is the equivalent base R step:  

```r
install.packages(".", repos=NULL)
```

where repos=NULL is used to install from download tar.gz or the source directory you may be in.

We already have `remotes` installed and need it so no need to change.

Reply:
https://github.com/single-cell-data/TileDB-SOMA/pull/2703/commits/3ae9824a28e713752f5ddc8d917c77631a3de8df